cmd/goose/main.go

```diff
@@ -36,11 +37,14 @@ var (
 	sslkey       = flags.String("ssl-key", "", "file path to SSL key in pem format (only support on mysql)")
 	noVersioning = flags.Bool("no-versioning", false, "apply migration commands with no versioning, in file order, from directory pointed to")
 	noColor      = flags.Bool("no-color", false, "disable color output (NO_COLOR env variable supported)")
+	timeout      = flags.Duration("timeout", 0, "duration that the migration should run for; e.g. 1h13m")
```

Comment:
Make this match whatever we decide in https://github.com/pressly/goose/pull/627/files#r1426157893